server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -908,7 +914,8 @@ export class AgenticChatController implements ChatHandlers {
 
         return {
             type: 'tool',
```

Comment:
I think this line makes it so that there's no padding in mynahUi. Can you please check how we can add padding without impacting the other tool cards?